R/prepare-admin-panel.R

```diff
@@ -917,7 +917,13 @@ prepare_admin_panel_components <- function(
 
   selected_session_data <- shiny::reactive({
     shiny::validate(shiny::need(selected_session(), label = "selected_session"))
-    selected_log_data() %>%
+    selected_data <- selected_log_data()
+
+    if (!"message" %in% names(selected_data)) {
+      selected_data$message <- NA
+    }
```

Comment:
This solves a problem with the analytics when displaying session details and there are no errors in database